### PR TITLE
use VERSION file for suffix when releasing

### PR DESCRIPTION
**Pull request by jakubgs**
Since the GitHub asset download is abysmally slow I want to upload those to DigitalOcean Spaces as well under the version from `VERSION` file when `RELEASE` is set to `true`.

This way we can avoid abysmally slow downloads, especially for iOS:
slow_ios

### CODE REVIEW

File: _assets/ci/Jenkinsfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ pipeline {
 
   parameters {
     booleanParam(
-      name: 'GITHUB_RELEASE',
+      name: 'RELEASE',
       description: 'Enable this to create a new GitHub release.',
```
Comment (pedropombeiro) [inline _assets/ci/Jenkinsfile]: Should the description also reflect that it is not only GitHub?

**Review by pedropombeiro — APPROVED** [review]

File: _assets/ci/Jenkinsfile (hunk 1 above)
Comment (jakubgs) [reply _assets/ci/Jenkinsfile]: Sure. Why not.

**Review by adambabik — APPROVED** [review]
LGTM

**Comment by adambabik** [comment]
@jakubgs 
> I want to upload those to DigitalOcean Spaces

I don't see any related code.

**Comment by jakubgs** [comment]
>I don't see any related code.

They are already uploaded, I just changed the suffix to use the contents of `VERSION` instead of timestamp + commit, this way `status-react` can download it using it's contents of `STATUS_GO_VERSION` file.

Also, this PR is not done because it fails due to a hardcoded name:
```log
+ make prepare-release
rm -rf /tmp/release-0.17.1-beta
mkdir -p /tmp/release-0.17.1-beta
mv build/bin/statusgo-android-16.aar /tmp/release-0.17.1-beta/status-go-android.aar
mv: cannot stat 'build/bin/statusgo-android-16.aar': No such file or directory
Makefile:219: recipe for target 'prepare-release' failed
make: *** [prepare-release] Error 1
```
https://ci.status.im/job/status-go/job/parallel/121/console

<blockquote><div><strong><a href="https://ci.status.im/job/status-go/job/parallel/121/console">status-go » parallel #121 Console [Jenkins]</a></strong></div></blockquote>

**Pull request merged**